tests/benchmark/performance/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+# Tests need to be added as executables first
+file(GLOB BENCHMARK_PERFORMANCE_TEST_LIST CONFIGURE_DEPENDS
+  "src/*.cpp"
+  "../custom_memory_manager/src/*.cpp"
+)
+
+set(target_name MeshKernelBenchmarkPerformanceTests)
+
+add_executable(${target_name} ${BENCHMARK_PERFORMANCE_TEST_LIST})
+
+target_include_directories(
+  ${target_name}
+  PRIVATE
+  include
+  ../custom_memory_manager/include
+)
+
+# Should be linked to the main library, as well as the google test library
+target_link_libraries(
+  ${target_name}
+  PRIVATE
+  MeshKernel
+  UtilsStatic
+  benchmark::benchmark
+  benchmark::benchmark_main
+)
+
+if (UNIX)
+   target_link_libraries(${target_name} PUBLIC ${CMAKE_DL_LIBS})
+endif (UNIX)
+
+# Make sure that coverage information is produced when using gcc
+if(PRODUCE_CODE_COVERAGE AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  set_target_properties( ${target_name} PROPERTIES COMPILE_FLAGS "-fprofile-arcs -ftest-coverage -static-libgcc -static-libstdc++")
+  set_target_properties(${target_name} PROPERTIES LINK_FLAGS "-ldl -lgcov --coverage")
+endif()
+
+# If you register a test, then ctest and make test will run it. You can also run
+# examples and check the output, as well. Command can be a target.
+add_test(NAME ${target_name}  COMMAND ${target_name} )
+
+# On Windows we bundle the dlls and then copy them to the target file directory
```

Comment:
in the benchmarking tests we might not need netcdf DLLs, if we do not read meshes from netcdf files

Reply:
I was ahead of myself here. I'll remove the custom command all together until it's needed.